data/json/npcs/militia/GM_Militia_Merchant.json

```diff
@@ -0,0 +1,228 @@
+[
+  {
+    "type": "npc",
+    "id": "MILITIA_MERCHANT",
+    "//": "A member of the Gray Moose Militia, trying to safe in their isolated location",
+    "name_unique": "Darby Sutton",
+    "class": "MILITIA_MERCHANT",
+    "attitude": 0,
+    "name_suffix": "Survivor",
+    "mission": 3,
+    "chat": "TALK_MILITIA_MAIN",
+    "faction": "GM_militia_faction"
+  },
+  {
+    "type": "npc_class",
+    "id": "MILITIA_MERCHANT",
+    "name": { "str": "Merchant" },
+    "job_description": "I'm a local shopkeeper.",
+    "traits": [ { "group": "NPC_starting_traits" }, { "group": "Appearance_demographics" } ],
+    "common": false,
+    "bonus_int": { "one_in": 4 },
+    "bonus_per": { "one_in": 4 },
+    "shopkeeper_item_group": "MILITIA_MERCHANT_misc",
+    "carry_override": "MILITIA_MERCHANT_misc",
+    "skills": [
+      {
+        "skill": "ALL",
+        "level": { "mul": [ { "one_in": 3 }, { "sum": [ { "dice": [ 2, 2 ] }, { "constant": -2 }, { "one_in": 4 } ] } ] }
+      },
+      { "skill": "mechanics", "bonus": { "one_in": 2 } },
+      { "skill": "electronics", "bonus": { "rng": [ 0, 2 ] } },
+      { "skill": "speech", "bonus": { "rng": [ 3, 5 ] } }
+    ]
+  },
+  {
+    "type": "item_group",
```

Comment:
If they want weapons and ammo why are they willing to give what they have away?
Bunging all the clothes (and guns/ammo if they're staying) into a seperate group to make them a single entry here would make it easier to tweak relative weights of stuff.

Reply:
I want to lock them down to trading only tools / supplies until you have a couple of quests achieved / traded with them enough that they trust you and have a replenished stock in their inventory. I'll have to implement that once I can see how it's done for Exodii trust.